src/unpacker/unpack.py

```diff
@@ -48,7 +50,16 @@ def unpack(
         for item in extracted_file_objects:
             current_fo.add_included_file(item)
 
-        current_fo.processed_analysis['unpacker'] = json.loads(Path(tmp_dir, 'reports', 'meta.json').read_text())
+        current_fo.processed_analysis['unpacker'] = sanitize_processed_analysis(
+            json.loads(
+                Path(
+                    tmp_dir,
+                    'reports',
+                    'meta.json',
+                ).read_text(),
+            ),
+        )
```

Comment:
Wow this is a rare case where breaking the lines actually worsens the readability :sweat_smile: 
```suggestion
        report = Path(tmp_dir) / 'reports/meta.json'
        current_fo.processed_analysis['unpacker'] = sanitize_processed_analysis(json.loads(report.read_text()))
```